fondant/cli.py

```diff
@@ -64,12 +71,25 @@ def run_data_explorer():
         f"{args.port}:8501",
         "--mount",
         f"type=bind,source={shlex.quote(args.data_directory)},target=/artifacts",
-        f"{shlex.quote(args.registry)}:{shlex.quote(args.tag)}",
     ]
 
+    if args.credentials:
+        cmd.extend(
+            [
+                "-v",
+                f"{args.credentials}:/root/.config/gcloud/application_default_credentials.json:ro",
```

Comment:
@GeorgesLorre This TARGET path is still hardcoded. How can we make it simple enough without having the user to search for which TARGET path they have to use for their Cloud Credentials? It's a similar problem for your PR (in the extra_volumes)